vspace/src/main/java/edu/asu/diging/vspace/web/staff/EditSlideExternalLinkController.java

```diff
@@ -0,0 +1,44 @@
+package edu.asu.diging.vspace.web.staff;
+
+import java.io.IOException;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import edu.asu.diging.vspace.core.exception.LinkDoesNotExistsException;
+import edu.asu.diging.vspace.core.exception.SlideDoesNotExistException;
+import edu.asu.diging.vspace.core.model.IExternalLinkSlide;
+import edu.asu.diging.vspace.core.services.ISlideExternalLinkManager;
+import edu.asu.diging.vspace.core.services.ISlideManager;
+
+@Controller
+public class EditSlideExternalLinkController extends EditSlideLinksController {
```

Comment:
`EditSlideLinksController` seems only to be used here; it doesn't seem like something that needs to be extracted in its own class.